### PR TITLE
Add admins to pipeline base & create scmUrl regex

### PR DESCRIPTION
**Pull request by d2lam**

### CODE REVIEW

File: models/pipeline.js [hunk 1]
```diff
@@ -53,7 +59,7 @@ module.exports = {
     update: Joi.object(mutate(MODEL, [
         'scmUrl'
     ], [
-        'configUrl'
+        'admins', 'configUrl'
```
Comment (stjohnjohnson) [inline models/pipeline.js]: Users should not be able to update admins

**Comment by stjohnjohnson** [comment]
LGTM otherwise. 👍 
excellent

File: models/pipeline.js (hunk 1 above)
Comment (d2lam) [inline models/pipeline.js]: Fixed!

**Comment by nkatzman** [comment]
👍

**Comment by jer** [comment]
We're just flying through those major versions 👍

**Pull request merged**